docs/application/native/guides/app-management/tidl.md

```diff
@@ -19,9 +19,13 @@ Usage:
   -o, --output=OUTPUT         The generated interface file.
```

Comment:
Should this be: 
Generates an interface file.

Reply:
Done. But, I don't add s(the third person singular) to "generate" verb.